cmd/osm-bootstrap/osm-bootstrap.go

```diff
@@ -11,6 +11,7 @@ import (
 	"os"
 	"time"
 
+	cmclient "github.com/jetstack/cert-manager/pkg/client/clientset/versioned"
```

Comment:
nit: this is a redirect to https://github.com/cert-manager/cert-manager

Reply:
pre-certmanager 1.8, the path is jetstack: https://github.com/cert-manager/cert-manager#importing-cert-manager-as-a-module